12-offer-encoding.md

```diff
@@ -463,6 +502,12 @@ the invoice request amount exceeds the amount it's expecting (i.e. its
 `offer_amount` after any currency conversion, multiplied by `invoice_request_quantity`, if
 any).
 
+Non-offer-response `invoice_requests` are currently required to
+explicitly state the `invoice_request_amount` in the chain currency,
+so `offer_amount` and `offer_currency` are redundant (but may be
+informative for the payer to know how the sender claims
+`invoice_request_amount` was derived).
+
 # Invoices
 
 Invoices are a payment request, and when the payment is made, 
```

Comment:
Outdated `offer_send_invoice` reference below and in the invoice's `types` section